src/fallback.rs

```diff
@@ -384,7 +384,7 @@ impl Span {
 
     #[cfg(procmacro2_semver_exempt)]
     pub fn source_file(&self) -> SourceFile {
-        CODEMAP.with(|cm| {
+        SOURCE_MAP.with(|cm| {
             let cm = cm.borrow();
             let fi = cm.fileinfo(*self);
```

Comment:
All the `cm` variables (here and elsewhere) could've been renamed to `sm` as well (but this is a very minor nitpick).